src/databricks/labs/ucx/source_code/known.json

```diff
@@ -21430,6 +21430,40 @@
     "ptyprocess.ptyprocess": [],
     "ptyprocess.util": []
   },
+  "pulp": {
```

Comment:
Same as for #2071, looks like a library that does not spark

Reply:
It's in scope with solacc, so there must be use case scenario.